### ISSUE TITLE
SAP R3 FunctionImport: Documentation Tag in Parameter fails the generation

### ISSUE THREAD

**Issue by sterlp** [oasis-tcs/odata-openapi#237]
Hello guys,

I am looking currently into a generation of a **FunctionImport** from an SAP system. I discovered that:

![image](https://user-images.githubusercontent.com/774984/223150015-0d887f48-80e8-4277-9454-4f588abe4371.png)

## Documentation would cause an error:
FunctionImport -> Parameter -> Documentation  = Error
but it works if it is used in EntityType ->  Parameter -> Documentation

```
C:\dev\workset\sap is-u api>odata-openapi3 -p -d --scheme https --skipBatchPath FunctionImport_SAP_R3.xml 
C:\Users\PAUL\AppData\Roaming\npm\node_modules\odata-openapi\node_modules\odata-csdl\lib\xml2json.js:2031
    throw enrichedException(e, xml, parser);
    ^

Error: Element Parameter, unexpected child: Documentation
    at checkElementAllowed (C:\Users\PAUL\AppData\Roaming\npm\node_modules\odata-openapi\node_modules\odata-csdl\lib\xml2json.js:1388:13)
    at parser.onopentag (C:\Users\PAUL\AppData\Roaming\npm\node_modules\odata-openapi\node_modules\odata-csdl\lib\xml2json.js:281:10)
    at emit (C:\Users\PAUL\AppData\Roaming\npm\node_modules\odata-openapi\node_modules\sax\lib\sax.js:624:35)       
    at emitNode (C:\Users\PAUL\AppData\Roaming\npm\node_modules\odata-openapi\node_modules\sax\lib\sax.js:629:5)
    at openTag (C:\Users\PAUL\AppData\Roaming\npm\node_modules\odata-openapi\node_modules\sax\lib\sax.js:825:5)
    at SAXParser.write (C:\Users\PAUL\AppData\Roaming\npm\node_modules\odata-openapi\node_modules\sax\lib\sax.js:1278:15)
    at module.exports.xml2json (C:\Users\PAUL\AppData\Roaming\npm\node_modules\odata-openapi\node_modules\odata-csdl\lib\xml2json.js:2029:12)
    at convert (C:\Users\PAUL\AppData\Roaming\npm\node_modules\odata-openapi\lib\cli.js:77:44)
    at Object.<anonymous> (C:\Users\PAUL\AppData\Roaming\npm\node_modules\odata-openapi\lib\cli.js:67:3)
    at Module._compile (node:internal/modules/cjs/loader:1159:14) {
  parser: { construct: '<Documentation>', line: 31, column: 27 }
}
```

## full XML example

```xml
<?xml version="1.0" encoding="utf-8"?>
<edmx:Edmx Version="1.0" xmlns:edmx="http://schemas.microsoft.com/ado/2007/06/edmx"
  xmlns:m="http://schemas.microsoft.com/ado/2007/08/dataservices/metadata"
  xmlns:sap="http://www.sap.com/Protocols/SAPData">
  <edmx:DataServices m:DataServiceVersion="2.0">
    <Schema Namespace="ZI_FOO_SRV" xml:lang="de" sap:schema-version="1"
      xmlns="http://schemas.microsoft.com/ado/2008/09/edm">

      <EntityType Name="Kontierungen_Liste" sap:content-version="1">
        <Key>
          <PropertyRef Name="Kontierung" />
        </Key>
        <Property Name="Massnahme" Type="Edm.String" MaxLength="12" sap:unicode="false"
          sap:label="Auftrag" sap:quickinfo="Auftragsnummer" sap:creatable="false"
          sap:updatable="false" sap:sortable="false" sap:filterable="false">
          <Documentation>
            <Summary>Schlüssel, der einen Auftrag innerhalb eines Mandanten eindeutig identifiziert.</Summary>
            <LongDescription />
          </Documentation>
        </Property>
      </EntityType>

      <EntityContainer Name="ZI_FOO_SRV_Entities" m:IsDefaultEntityContainer="true"
        sap:supported-formats="atom json xlsx">
        <FunctionImport Name="KontierungEckendeSetzen" ReturnType="ZI_FOO_SRV.Kontierungen_Liste"
          EntitySet="Kontierungen_ListeSet" m:HttpMethod="POST"
          sap:action-for="ZI_FOO_SRV.Kontierungen_Liste">
          <Parameter Name="Eckende" Type="Edm.DateTime" Mode="In" Precision="0">
            <!-- this Documentation block causes an exception -->
            <Documentation>
              <Summary>Zeitstempel</Summary>
              <LongDescription />
            </Documentation>
          
          </Parameter>
          <Parameter Name="Kontierung" Type="Edm.String" Mode="In" MaxLength="12" />
          <Parameter Name="Massnahme" Type="Edm.String" Mode="In" MaxLength="12" />
        </FunctionImport>
      </EntityContainer>
      <atom:link rel="self"
        href="https://sap_r3.de/sap/opu/odata/SAP/ZI_FOO_SRV/$metadata"
        xmlns:atom="http://www.w3.org/2005/Atom" />
      <atom:link rel="latest-version"
        href="https://sap_r3.de/sap/opu/odata/SAP/ZI_FOO_SRV/$metadata"
        xmlns:atom="http://www.w3.org/2005/Atom" />
    </Schema>
  </edmx:DataServices>
</edmx:Edmx>
```
I am not an export in XSD conversion -- issue happens in the **cli.js** **csdl.xml2json(text)**

Any help is welcome.

Thanks

Paul

**Comment by sterlp** [comment]
Parameters aren't an issue -- found it, sorry for that.

**Comment by ralfhandl** [comment]
@sterlp Could you please check PR #238?

Thanks in advance

**Comment by sterlp** [comment]
@ralfhandl works like a charme

![image](https://user-images.githubusercontent.com/774984/223158996-c690b761-19d5-4e95-b00d-47f250bb612c.png)

**Comment by ralfhandl** [comment]
Fix published as [`odata-openapi@0.21.2`](https://www.npmjs.com/package/odata-openapi)
